_protected/app/system/modules/user/forms/processing/LoginFormProcess.php

```diff
@@ -76,7 +76,7 @@ public function __construct()
                 $oSecurityModel->addLoginLog(
                     $sEmail,
                     'Guest',
-                    $sPassword,
+                    '*****',
```

Comment:
Makes sense, although it's an incorrect password, no one wants it to be stored. This was to prevent bad users/hackers from trying to guess the read passwords of other users. But we don't want our password to be stored when we do a small typo.

Reply:
Since the repo where you made this change has been deleted, I will have to do it for you. Anyway, I really appreciate this change :)

Reply:
For the record, this has been applied here: https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/22e978e207566a756b5fb5a519c5beff8c82cd3e

Reply:
FYI, for the moment, I still leave the wrong password cleared for the admin form. Since it could be useful to track hackers trying to login to the admin dashboard.
Anyway, if you also want to replace it by asterisks, you can always do it on that line https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/master/_protected/app/system/modules/admin123/forms/processing/LoginFormProcess.php#L74